src/util/StreamerMode.cpp

```diff
@@ -32,51 +42,108 @@ bool isInStreamerMode()
             return true;
         case StreamerModeSetting::Disabled:
             return false;
-    }
+        case StreamerModeSetting::DetectObs:
 
-#ifdef USEWINSDK
-    if (!IsWindowsVistaOrGreater())
-    {
-        return false;
-    }
-    static bool cache = false;
-    static QDateTime time = QDateTime();
+#ifdef Q_OS_LINUX
 
-    if (time.isValid() &&
-        time.addSecs(cooldownInS) > QDateTime::currentDateTime())
-    {
-        return cache;
-    }
+            static bool cache = false;
+            static QDateTime time = QDateTime();
+
+            if (time.isValid() &&
+                time.addSecs(cooldownInS) > QDateTime::currentDateTime())
+            {
+                return cache;
+            }
+            time = QDateTime::currentDateTime();
 
-    time = QDateTime::currentDateTime();
+            QProcess p;
+            p.start("grep", {"-x", broadcastingBinaries().join("|")},
+                    QIODevice::NotOpen);
 
-    WTS_PROCESS_INFO *pWPIs = nullptr;
-    DWORD dwProcCount = 0;
+            if (p.waitForFinished(1000))
```

Comment:
[exitCode value is not valid unless exitStatus() returns NormalExit](https://doc.qt.io/qt-5/qprocess.html#exitCode)
```suggestion
            if (p.waitForFinished(1000) && p.exitStatus() == QProcess::NormalExit)
```